fixtures/tests/bootstrap.js

```diff
@@ -0,0 +1,31 @@
+import puppeteer from 'puppeteer';
+import { expect } from 'chai';
+import * as utils from '../flows/utils';
+
+
+// puppeteer options
+const opts = {
+    headless: false,
+    //slowMo: 100,
+    //timeout: 10000
```

Comment:
Is this a WIP or something that can be removed / turned into comments?

Reply:
fixed